tools/print_tock_memory_usage.py

```diff
@@ -277,29 +287,26 @@ def print_groups(title, groups):
         group_sum = group_sum + group_size
 
     print(title + ": " + str(group_sum) + " bytes")
-    print(output, end = ' ')
+    print(output, end = '')
 
 def print_symbol_information():
     """Print out all of the variable and function groups with their flash/RAM
        use."""
     variable_groups = {}
-    group_symbols(variable_groups, kernel_initialized, show_waste, "RAM")
-    group_symbols(variable_groups, kernel_uninitialized, show_waste, "Flash+RAM")
-    if (show_waste):
-        print() # Place an newline after waste reports
-
+    gaps = group_symbols(variable_groups, kernel_initialized, show_waste, "RAM")
+    gaps = gaps + group_symbols(variable_groups, kernel_uninitialized, show_waste, "Flash+RAM")
```

Comment:
Copying my one comment from the other PR, should this be just "RAM" and not "Flash+RAM"? It seems like wasted flash is printed elsewhere.